packages/wp-now/project.json

```diff
@@ -2,7 +2,7 @@
 	"name": "wp-now",
 	"$schema": "../../node_modules/nx/schemas/project-schema.json",
 	"sourceRoot": "packages/wp-now/src",
-	"projectType": "application",
+	"projectType": "library",
```

Comment:
cc @sejas  – there's a bunch of wp-now changes in this PR so that I could import and use `startServer()`